### PR TITLE
Pr/misc

### PR DESCRIPTION
**Pull request by pavanbalaji**

### CODE REVIEW

File: src/mpid/ch4/include/mpid_ticketlock.h [hunk 1]
```diff
@@ -24,14 +24,25 @@ typedef union MPIDI_CH4_Ticket_lock {
 MPL_STATIC_INLINE_PREFIX void MPIDI_CH4I_Thread_mutex_acquire(MPIDI_CH4_Ticket_lock * m)
 {
     uint16_t u = __sync_fetch_and_add(&m->s.clients, 1);
+
+    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPID_REQUEST_CREATE_HOOK);
+    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPID_REQUEST_CREATE_HOOK);
```
Comment (raffenet) [inline src/mpid/ch4/include/mpid_ticketlock.h]: This doesn't look right.

File: src/mpid/ch4/include/mpid_ticketlock.h [hunk 2]
```diff
@@ -40,39 +51,64 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_CH4I_Thread_mutex_try_acquire(MPIDI_CH4_Ticke
     uint16_t u2 = u + 1;
     uint32_t val = ((uint32_t) u << 16) + u;
     uint32_t val2 = ((uint32_t) u2 << 16) + u;
+    int ret = EBUSY;
+
+    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPID_REQUEST_DESTROY_HOOK);
```
Comment (raffenet) [inline src/mpid/ch4/include/mpid_ticketlock.h]: Not right.

**Comment by pavanbalaji** [comment]
Force pushed some changes to fix these errors (it turned out to be an error in my script).  I also added another commit to fixup copyright issues.

**Comment by raffenet** [comment]
test:jenkins/ch4/all

**Pull request merged**